updates governance model without members (#1346)
publiccodenet/about: organization/governance-model.md

## Overview

The Foundation for Public Code is a [non-profit association (vereniging) under Dutch law](https://business.gov.nl/starting-your-business/choosing-a-business-structure/association/).

> The association's members are the regional Foundation for Public Code chapter organizations.

Reviewer: This is not yet true, we should not merge this wording until it is.

Author: Yes, agreed

These members are responsible for and steer the association's operations, including priorities and annual budget for the organization.

## Foundation bodies

> The Foundation consists of four bodies:

Reviewer 2: ```suggestion
The Foundation consists of five bodies:
```
With the addition of the strategic council to the list, we should +1 here.

Reviewer 2: (I am assuming the resolve without comment was a miss-click.)

Reviewer 3: It's fixed!

* approves the Foundation's operational plan for the coming year
* elects the board of directors

These decisions are agreed through a majority vote.

> Each member has 3 votes.

Reviewer 2: This is perhaps over-simplifying it. We wanted the representatives to act individually, but this implies that they act in unison. (But also, perhaps that level of detail is not vital here, but it is enough that it is spelled out in the bylaws.)

Reviewer 3: @clausmullie and I envisioned this as an overview of the governance for a brand new staff member. In that context, would you appreciate that level of detail?

Reviewer 2: I think this is fine enough in that regard.

This ensures democratic ownership and control by all members.
Additionally, the members of the strategic council each have 1 vote (see below).

The general assembly meets at least once every year.
Minutes from the general assembly meetings are shared with all members and [publicly archived](general-assemblies/index.md).

We commit to holding all meaningful votes electronically, so that all members can be represented regardless of ability to attend physically.

### Board of directors

The board of directors is elected by the general assembly.
By law, the responsibilities include that of the chair, secretary and treasurer.
The general assembly can add responsibilities and define how they are divided into roles within the board of directors.

The board of directors proposes the Foundation's policy plan, operational plan and budgeting for the coming year.
These decisions are agreed upon through majority vote.

The board of directors meets regularly and is required to keep the general assembly informed about its activities.
Minutes are [publicly archived](board-of-directors-meetings/index.md).

The current board members are:

* Ben Cerveny (Chair)
* Richard Pope

### Strategic council

> The [strategic council](strategic-council.md) helps the board of directors with key strategic decisions as well as serve as ambassadors for its mission.

Reviewer 3: ```suggestion
The [strategic council](strategic-council.md) helps the board of directors with key strategic decisions and serve as ambassadors for our mission.
```
